### ISSUE TITLE
CKEditorError: ckeditor-duplicated-modules

### ISSUE THREAD

**Issue by ArkIv** [ckeditor/ckeditor5-vue#111]
>**Does not work ckeditor5-editor-classic, writes:**
CKEditorError: ckeditor-duplicated-modules: Some CKEditor 5 modules are duplicated. Read more: . . . .
```javascript
import CKEditor from "@ckeditor/ckeditor5-vue";
    // ⚠️ NOTE: We don't use @ckeditor/ckeditor5-build-classic any more!
    // Since we're building CKEditor from source, we use the source version of ClassicEditor.
    //import InlineEditor from "@ckeditor/ckeditor5-editor-inline/src/inlineeditor";_
import ClassicEditor from "@ckeditor/ckeditor5-editor-classic/src/classiceditor";
. . . . . 
 components: {
    ckeditor: CKEditor.component
  },
data() {
    return {
      editor: ClassicEditor, //InlineEditor,
 }
. . . . . 
```
> ## ***And here is ckeditor5-editor-inline, everything works normally. :+1:***

**Comment by Mgsy** [comment]
Hi, can you share your `package.json`? There's a chance that you have some old dependencies.

**Comment by ArkIv** [comment]
But, ckeditor5-editor-inline, everything works normally.
Can "^ 16.0.0"?
the same. ckeditor-duplicated-modules  @ckeditor/ckeditor5-editor-balloon": "^16.0.0  
```
 "dependencies": {
    "@ckeditor/ckeditor5-alignment": "^15.0.0",
    "@ckeditor/ckeditor5-autoformat": "^15.0.0",
    "@ckeditor/ckeditor5-basic-styles": "^15.0.0",
    "@ckeditor/ckeditor5-block-quote": "^15.0.0",
    "@ckeditor/ckeditor5-dev-utils": "^12.0.5",
    "@ckeditor/ckeditor5-dev-webpack-plugin": "^8.0.5",
    "@ckeditor/ckeditor5-editor-balloon": "^16.0.0",
    "@ckeditor/ckeditor5-editor-classic": "^16.0.0",
    "@ckeditor/ckeditor5-editor-inline": "^15.0.0",
    "@ckeditor/ckeditor5-essentials": "^15.0.0",
    "@ckeditor/ckeditor5-font": "^15.0.0",
    "@ckeditor/ckeditor5-heading": "^15.0.0",
    "@ckeditor/ckeditor5-image": "^15.0.0",
    "@ckeditor/ckeditor5-link": "^15.0.0",
    "@ckeditor/ckeditor5-list": "^15.0.0",
    "@ckeditor/ckeditor5-media-embed": "^15.0.0",
    "@ckeditor/ckeditor5-paragraph": "^15.0.0",
    "@ckeditor/ckeditor5-paste-from-office": "^15.0.0",
    "@ckeditor/ckeditor5-table": "^15.0.0",
    "@ckeditor/ckeditor5-theme-lark": "^15.0.0",
    "@ckeditor/ckeditor5-upload": "^15.0.0",
    "@ckeditor/ckeditor5-vue": "^1.0.0",
    "core-js": "^3.4.3",
    "postcss-loader": "^3.0.0",
    "raw-loader": "^0.5.1",
    "vue": "^2.6.10",
    "vue-ckeditor5": "^0.4.1",
    "vue-router": "^3.1.3",
    "vuex": "^3.1.2"
  },
  "devDependencies": {
    "@vue/cli-plugin-babel": "^4.1.0",
    "@vue/cli-plugin-eslint": "^4.1.0",
    "@vue/cli-plugin-router": "^4.1.0",
    "@vue/cli-plugin-vuex": "^4.1.0",
    "@vue/cli-service": "^4.1.0",
    "@vue/eslint-config-prettier": "^5.0.0",
    "babel-eslint": "^10.0.3",
    "eslint": "^5.16.0",
    "eslint-plugin-prettier": "^3.1.1",
    "eslint-plugin-vue": "^5.0.0",
    "node-sass": "^4.12.0",
    "prettier": "^1.19.1",
    "sass-loader": "^8.0.0",
    "vue-template-compiler": "^2.6.10"
  }
```

**Comment by Mgsy** [comment]
As we introduced a new versioning policy, right now, every package has the same version. As you can see, all your dependencies are `15.0.0`, including `ckeditor5-editor-inline` - that's why it works. Classic and balloon editors are `16.0.0`, so are not compatible with the rest of the packages. 

Please, update all your dependencies to `16.0.0`, which is the latest version released yesterday (or stay with `15.0.0`) and everything should work fine.

**Comment by ArkIv** [comment]
Yes! works. How at me so it turned out, put with a difference of an hour.:)
Thanks.
